api/helpers/gamestate/moves/pass/validate.js

```diff
@@ -0,0 +1,48 @@
+const GamePhase = require('../../../../../utils/GamePhase.json');
+
+module.exports = {
+  friendlyName: 'Validate request to Pass',
+
+  description: 'Verifies whether a request to pass a move is legal, throwing explanatory error if not.',
+
+  inputs: {
+    currentState: {
+      type: 'ref',
+      descriptions: 'Object containing the current game state',
+      required: true,
+    },
+    /**
+     * @param { MoveType.PASS } requestedMove.moveType - Specifies that this a passing move
+     * @param { Object } requestedMove - Object describing the request to pass
```

Comment:
The object param should go above the individual attributes since they describe its shape
```suggestion

     * @param { Object } requestedMove - Object describing the request to pass
     * @param { MoveType.PASS } requestedMove.moveType - Specifies that this a passing move
```